src/layouts/Header/MenuSearch/index.tsx

```diff
@@ -30,8 +32,12 @@ export const MenuSearch = () => {
   const debouncedSearchTerm = useDebounce(searchValue, 500);
 
   useEffect(() => {
+	dispatch(setIsNew(false)) 
     !(searchValue.trim() == "") &&
-      dispatch(getBooks(1, searchValue.replace(/\s/g, "").toLowerCase())) &&
+      dispatch(setIsLoading(true)) &&
+      dispatch(
```

Comment:
Why u use && here? 
Seems to me u can just write all those function call in a row.